migrate-to-github/README.md

```diff
@@ -0,0 +1,77 @@
+Check the [what-to-migrate.md]() file first, please.
+
+### How we want to do it
+
+We will use the XXX tool (not released yet).
+  - still thinking about its name :D I think about two possibilities:
+    - `volagit` or `forgit` -> these are just random names containing `git`... I think
+       about them because they sounds cool :D
```

Comment:
Not sure what the tool will do and in what scope. But if it will be specifically for migrating from Pagure to GitHub, I would go with `pagure2github`, or if there is an ambition to support multiple forges in the future, I would go with `git-forge-migrate` or something else that can be easily googled.

Reply:
Good point with pagure2github.  There were reasons for not just using it.  Do you remember the reasons?

Reply:
He, actually: https://github.com/SilleBille/pagure2github

Reply:
Heh, cool.
But it's weird. When I try to google "pagure2github", it doesn't find the repo.

Reply:
That's indeed weird, duckduckgo places the project as the first result.